tests/testthat/test-unnecessary_lambda_linter.R

```diff
@@ -12,6 +12,9 @@ test_that("unnecessary_lambda_linter skips allowed usages", {
   # at any nesting level
   expect_lint("parLapply(cl, x, function(xi) foo(xi, 2, bar(baz(xi))))", NULL, linter)
 
+  # multi-argument case
+  expect_lint("lapply(l, function(x, y) setdiff(x, y))", NULL, linter)
```

Comment:
oh I see now. I wouldn't add this test. the code as written can't run (y has no default), and any working version can be replaced without lambda.

no sense adding an incorrect test

Reply:
Can replace it with this one:

``` r
l <- list("x" = 1:3, "y" = c(1:3, NA))

lapply(l, function(x, na.rm = TRUE) mean(x, na.rm = na.rm))
#> $x
#> [1] 2
#> 
#> $y
#> [1] 2
```

<sup>Created on 2022-10-03 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>